org-code-javabuilder/lib/src/main/java/org/code/neighborhood/Painter.java

```diff
@@ -0,0 +1,144 @@
+package org.code.neighborhood;
+
+public class Painter {
+    private int xLocation;
+    private int yLocation;
+    private Direction direction;
+    private int remainingPaint;
```

Comment:
Just curious, do we track the color of the paint that the painter has picked up?  Or is there only one color of paint?

Reply:
Conceptually, the paint coloring is "magic" (I believe that's the official term 🙂) - it doesn't have a color associated with it until it is used to paint a square.

Reply:
Love it!  My kids want to paint their rooms, where can I get some of this paint?  :-)